### PR TITLE
Typos in StudentProfileAttributesTest, CoursesLogicTest and InstructorCoursesPageDataTest #6991

### LINKED ISSUE #6991: Typos in StudentProfileAttributesTest, CoursesLogicTest and InstructorCoursesPageDataTest
StudentProfileAttributesTest.java
Invalidty is not a word according to provided dictionary
``` java
testGetInvalidtyInfoForValidProfileWithEmptyValues();
```

CoursesLogicTest.java
Cousre is not a word according to provided dictionary
``` java
        CourseAttributes notSampleCousre = new CourseAttributes("course.id", "not sample course", "UTC");
```

InstructorCoursesPageDataTest.java
Withour is not a word according to provided dictionary
``` java
InstructorCoursesPageData pageData = new InstructorCoursesPageData(instructorAccountWithourCourses);
```

### PR DESCRIPTION
**Pull request by wkk91193**
Fixes #6991

**Outline of Solution**
 Removed simple typos in  StudentProfileAttributesTest,CoursesLogicTest and InstructorCoursesPageDataTest

### CODE REVIEW

**Comment by leeyimin** [comment]
Hi @wkk91193! Thanks for contributing to TEAMMATES! 
It seems that you are currently submitting PRs for two first-timers issues so you may have to choose to close one of them. Do take note to follow the guidelines for the PR title and description next time!

File: .settings/org.eclipse.wst.validation.prefs [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 USER_BUILD_PREFERENCE=enabledBuildValidatorListorg.eclipse.wst.wsi.ui.internal.WSIMessageValidator;
 USER_MANUAL_PREFERENCE=enabledManualValidatorListorg.eclipse.wst.wsi.ui.internal.WSIMessageValidator;
-USER_PREFERENCE=overrideGlobalPreferencestruedisableAllValidationfalseversion1.2.402.v201212031633
+USER_PREFERENCE=overrideGlobalPreferencestruedisableAllValidationfalseversion1.2.700.v201508251749
 eclipse.preferences.version=1
 override=true
 suspend=false
+vals/org.eclipse.jst.jsp.core.JSPBatchValidator/global=TF01
+vals/org.eclipse.jst.jsp.core.JSPContentValidator/global=TF01
+vals/org.eclipse.wst.html.core.HTMLValidator/global=TF01
```
Comment (leeyimin) [inline .settings/org.eclipse.wst.validation.prefs]: These changes should be removed.

File: src/test/java/teammates/test/cases/logic/CoursesLogicTest.java [hunk 2]
```diff
@@ -1,7 +1,6 @@
 package teammates.test.cases.logic;
 
 import static teammates.common.util.Const.EOL;
-
```
Comment (leeyimin) [inline src/test/java/teammates/test/cases/logic/CoursesLogicTest.java]: This change should be removed as well.

**Comment by wkk91193** [comment]
@leeyimin  Yes I fixed the remaining typo instructorAccountWithourCourses!

File: src/test/java/teammates/test/cases/logic/CoursesLogicTest.java (hunk 2 above)
Comment (wkk91193) [inline src/test/java/teammates/test/cases/logic/CoursesLogicTest.java]: @leeyimin  Fixed!

File: .settings/org.eclipse.wst.validation.prefs (hunk 1 above)
Comment (wkk91193) [inline .settings/org.eclipse.wst.validation.prefs]: @leeyimin Fixed! 
 Line 4 : USER_PREFERENCE=overrideGlobalPreferencestruedisableAllValidationfalseversion1.2.700.v201508251749
changed to  USER_PREFERENCE=overrideGlobalPreferencestruedisableAllValidationfalseversion1.2.402.v201212031633

Removed lines  8,9,10

File: .settings/org.eclipse.wst.validation.prefs [hunk 3]
```diff
@@ -5,5 +5,4 @@ USER_PREFERENCE=overrideGlobalPreferencestruedisableAllValidationfalseversion1.2
 eclipse.preferences.version=1
 override=true
 suspend=false
-vals/org.eclipse.wst.xml.core.xml/global=FF03
-vf.version=3
+
```
Comment (leeyimin) [inline .settings/org.eclipse.wst.validation.prefs]: These changes as well!

File: src/test/java/teammates/test/cases/logic/CoursesLogicTest.java (hunk 2 above)
Comment (leeyimin) [inline src/test/java/teammates/test/cases/logic/CoursesLogicTest.java]: Hmm as these lines are not related to the current issue, they should not be changed at all. You should actually reinsert the newlines you removed between lines 4 to 10.

File: src/test/java/teammates/test/cases/logic/CoursesLogicTest.java [hunk 4]
```diff
@@ -1,13 +1,10 @@
...
 import java.util.HashMap;
 import java.util.List;
-
```
Comment (leeyimin) [inline src/test/java/teammates/test/cases/logic/CoursesLogicTest.java]: There should not be a change here as well.

File: src/test/java/teammates/test/cases/logic/CoursesLogicTest.java [hunk 5]
```diff
@@ -1,13 +1,10 @@
...
 import org.apache.commons.lang3.StringUtils;
 import org.testng.annotations.Test;
-
```
Comment (leeyimin) [inline src/test/java/teammates/test/cases/logic/CoursesLogicTest.java]: Same here.

**Comment by wkk91193** [comment]
@leeyimin  Hope the issue is resolved! My bad, in my previous commit the file  .settings/org.eclipse.wst.validation.prefs had been committed without my knowledge, though file had been already excluded in the .gitignore file. Pretty weird!
